.github/workflows/semgrep.yml

```diff
@@ -2,15 +2,15 @@ name: Semgrep
 
 on:
   # Scan changed files in PRs (diff-aware scanning):
-#   pull_request: {}
+  pull_request: {}
```

Comment:
If we're not running this on PRs, can you delete this line?

Reply:
I want to run it on PRs